docs/index.rst

```diff
@@ -0,0 +1,37 @@
+.. pVAC-Seq documentation master file, created by
+   sphinx-quickstart on Wed Aug 31 08:31:01 2016.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+pVAC-Seq
+====================================
+
+pVAC-Seq is a cancer immunotherapy pipeline for the identification of **p**\ ersonalized **V**\ ariant **A**\ ntigens by **C**\ ancer **Seq**\ uencing (pVAC-Seq) that integrates tumor mutation and expression data (DNA- and RNA-Seq). It enables cancer immunotherapy research by using massively parallel sequence data to predicting tumor-specific mutant peptides (neoantigens) that can elicit anti-tumor T cell immunity. It is being used in studies of checkpoint therapy response and to identify targets for cancer vaccines and adoptive T cell therapies. For more general information, see the [manuscript published in Genome Medicine](http://www.genomemedicine.com/content/8/1/11)
+
```

Comment:
I'm not sure I did the above link correctly - can someone review and fix, if necessary?